python-ecosys/requests/manifest.py

```diff
@@ -1,3 +1,3 @@
-metadata(version="0.10.2", pypi="requests")
+metadata(version="0.10.3", pypi="requests")
```

Comment:
Please leave the version unchanged.

The change to the code is superficial, and the generated .mpy file is equivalent, so no need to bump the version.

Reply:
reverted